tests/call_stack/src/lib.rs

```diff
@@ -0,0 +1,50 @@
+#![no_std]
+use soroban_sdk::{contractimpl, symbol, vec, BytesN, Env};
+
+pub struct OuterContract;
+
+#[contractimpl]
+impl OuterContract {
+    pub fn outer(env: Env, contract_id: BytesN<32>) {
+        env.invoke_contract(&contract_id, &symbol!("inner"), vec![&env])
```

Comment:
Could you add an assertion in the out contract on what it sees the call stack is? I assume it will be a length one stack with just itself in it?

Reply:
Great idea. Done.